actors/miner/tests/util.rs

```diff
@@ -2140,15 +2141,32 @@ impl ActorHarness {
                 }
             }
 
+            let state = self.get_state(&rt);
+
+            // Advance to the end of the deadline. We do this manually because we need to be at the
+            // end of the deadline to figure out whether or not we take the fees from vested or
+            // unvested funds.
+            assert!(state.deadline_cron_active, "deadline is active");
+            let deadline = new_deadline_info_from_offset_and_epoch(
+                &rt.policy,
+                state.proving_period_start,
+                *rt.epoch.borrow(),
+            );
+            rt.set_epoch(deadline.last());
+
+            let expected_enrollment = deadline.last() + rt.policy.wpost_challenge_window;
             let unvested = unvested_vesting_funds(rt, &state);
-            let available = rt.get_balance() + unvested.clone() - &state.initial_pledge;
-            let burnt_funds =
-                std::cmp::max(&TokenAmount::zero(), std::cmp::min(&available, &daily_fee)).clone();
-            let pledge_delta = std::cmp::min(&unvested, &daily_fee).neg().clone();
-            let cfg = CronConfig { burnt_funds, pledge_delta, ..Default::default() };
+            let immediately_vesting = immediately_vesting_funds(rt, &state);
+            let available_to_burn = rt.get_balance() - &state.initial_pledge;
+            let burnt_funds = daily_fee.clone().clamp(TokenAmount::zero(), available_to_burn);
```

Comment:
`clamp` is a generally useful function for handling min/max cases like this.

Reply:
oh, that's nice :ok_hand: